src/styles/submit.styles.js

```diff
@@ -0,0 +1,178 @@
+import styled from "styled-components";
+import { GREEN, P, H3, CREAM, BLUE, YELLOW, BLACK, fonts } from "./styles";
+import { min } from "./breakpoints";
+import Cursor from "../images/cursor-pointer.svg";
+import { ArrowUpCircle } from "react-feather";
+
+export const HeaderImage = styled.img`
+  width: 100%;
+`;
+
+export const ContentContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  ${min.desktop} {
+    flex-direction: row;
+  }
+`;
+
+export const InformationContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  ${min.desktop} {
+    width: 33%;
+  }
+`;
+
+export const FormContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  ${min.tablet} {
+    margin-left: 12.5%;
+    width: 75%;
+  }
+  ${min.desktop} {
+    margin-left: 8.33%;
+    width: 50%;
+  }
+`;
+
+export const Input = styled.input`
+  type: text;
+  background: ${BLACK};
+  border: solid ${CREAM};
+  color: ${CREAM};
+  padding: 4%;
+  font-family: ${fonts.IBMPlexMono};
+  font-size: 1.125rem;
+  line-height: 1.375rem;
+  margin-bottom: 1rem;
+  width: 92%;
+  outline: none;
```

Comment:
@mnav0 might know better than I do but I think outline might be necessary for accessibility? i think it's fine to leave it as the default blue